gdcdictionary/schemas/aligned_reads_index.yaml

```diff
@@ -10,6 +10,7 @@ project: '*'
 description: "Data file containing the index for a set of aligned reads."
 additionalProperties: false
 submittable: true
+downloadable: false
```

Comment:
We messed up, ticket updated, but this & somatic_mutation_index need to be downloadable.